src/routes/activitypub/nodeinfo.js

```diff
@@ -25,12 +25,15 @@ router.get("/", async function (req, res) {
     const bookmarksDb = req.app.get("bookmarksDb");
     let bookmarkCount = await bookmarksDb.getBookmarkCount();
 
+    const repository = process.env.REPOSITORY || 'https://github.com/ckolderup/postmarks';
+
     // TODO: activeMonth and activeHalfyear should be dynamic, currently static
     let nodeInfo = {
       version: 2.0,
       software: {
         name: instanceType,
         version: instanceVersion,
+        repository
```

Comment:
so technically I believe `repository` is part of the `2.1` schema (according to http://nodeinfo.diaspora.software/schema.html), and would also need a key name of `repository` here.

Reply:
I'm not an expert on it either. Should we update to 2.1, support both, or do nothing? This is a very minor feature. It allows discovery of the repository (and the FEDERATION.md) but I don't know if anything is doing the discovery at this point. I don't mind withdrawing the PR if that's the decision.

Reply:
I just threw in the 2.0 nodeinfo to align with Mastodon (which I'm marginally more familiar with). We could make it a more 2.1-compliant thing... I think the main services calling this endpoint tend to be things like fedidb which are tracking Fediverse growth, but I could be wrong.